wear/src/main/kotlin/au/com/shiftyjelly/pocketcasts/wear/networking/PocketCastsNetworkingRules.kt

```diff
@@ -0,0 +1,93 @@
+package au.com.shiftyjelly.pocketcasts.wear.networking
+
+import androidx.annotation.VisibleForTesting
+import au.com.shiftyjelly.pocketcasts.BuildConfig
+import com.google.android.horologist.networks.ExperimentalHorologistNetworksApi
+import com.google.android.horologist.networks.data.NetworkInfo
+import com.google.android.horologist.networks.data.NetworkStatus
+import com.google.android.horologist.networks.data.NetworkType
+import com.google.android.horologist.networks.data.Networks
+import com.google.android.horologist.networks.data.RequestType
+import com.google.android.horologist.networks.data.RequestType.MediaRequest.Companion.DownloadRequest
+import com.google.android.horologist.networks.data.RequestType.MediaRequest.Companion.LiveRequest
+import com.google.android.horologist.networks.rules.Allow
+import com.google.android.horologist.networks.rules.NetworkingRules
+import com.google.android.horologist.networks.rules.RequestCheck
+import timber.log.Timber
+
+object PocketCastsNetworkingRules : NetworkingRules {
+    override fun isHighBandwidthRequest(requestType: RequestType): Boolean {
+        if (BuildConfig.DEBUG) {
+            // For testing purposes fail if we get unknown requests
+            check(requestType != RequestType.UnknownRequest) {
+                "Unknown request type. Failing on debug builds."
+            }
```

Comment:
This seemed like a good way to make sure we don't use these networking rules but forget to set a request type on the request.